### PR TITLE
[ISSUE-2040] shift+enter for new line, enter for submit (message)

### PR DESCRIPTION
**Pull request by wperingbart**
### Description:

- It was to make new line for shift+enter and submission for enter only.
- Solution was to check if shift was pressed on keypress event (and otherwise make submission).

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [x] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [x] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

**Comment by franckc** [comment]
@wperingbart Thank you for working on this ! :)
I did a quick test on my laptop (MacOS 10.14.5, Chrome 76.0) and I'm seeing the following behavior:
 - Typing Enter submits the message. That's what we want.
 - Typing Shift Enter submits the message. I think we want to create a new line.
 - Typing Ctrl Enter does not do anything. I think we want to create a new line.

**Comment by wperingbart** [comment]
hi @franckc 
I fixed it.
Now it reacts with alt+enter also (new line)
Please check it out.

File: dapps/marketplace/src/utils/pasteIntoInput.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default function pasteIntoInput(el, text) {
+  el.focus()
+  if (
+    typeof el.selectionStart == 'number' &&
```
Comment (franckc) [inline dapps/marketplace/src/utils/pasteIntoInput.js]: nit: we use the === operator throughout our codebase. to be consistent, could you update ?
Same comment fo line 11 below.

File: dapps/marketplace/src/utils/pasteIntoInput.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export default function pasteIntoInput(el, text) {
```
Comment (franckc) [inline dapps/marketplace/src/utils/pasteIntoInput.js]: nit: could you document what this new method does using JSDoc style comments ?

File: dapps/marketplace/src/utils/pasteIntoInput.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  el.focus()
+  if (
+    typeof el.selectionStart == 'number' &&
+    typeof el.selectionEnd == 'number'
+  ) {
+    const val = el.value
+    const selStart = el.selectionStart
+    el.value = val.slice(0, selStart) + text + val.slice(el.selectionEnd)
+    el.selectionEnd = el.selectionStart = selStart + text.length
+  } else if (typeof document.selection != 'undefined') {
```
Comment (franckc) [inline dapps/marketplace/src/utils/pasteIntoInput.js]: nit: add a short comment either here in the description of the pasteIntoInput method why in some cases el.selectionStart/End would not be defined (I assume it is not supported by all browsers ?).

**Comment by wperingbart** [comment]
hi @franckc 
I updated it
Would you please review again?

**Comment by franckc** [comment]
@wperingbart I don't see any changes to the CSS ? Did you perhaps forget to commit those changes before pushing ? Also please do not commit changes to packages/contracts/build/contracts - these are auto-generated files by @origin/services when docker-compose starts...

**Comment by wperingbart** [comment]
@franckc 
I added comments there.
Please check it

File: dapps/marketplace/src/utils/pasteIntoInput.js (hunk 1 above)
Comment (wperingbart) [reply dapps/marketplace/src/utils/pasteIntoInput.js]: done

File: dapps/marketplace/src/utils/pasteIntoInput.js (hunk 2 above)
Comment (wperingbart) [reply dapps/marketplace/src/utils/pasteIntoInput.js]: done

File: dapps/marketplace/src/utils/pasteIntoInput.js (hunk 3 above)
Comment (wperingbart) [reply dapps/marketplace/src/utils/pasteIntoInput.js]: done

**Comment by wperingbart** [comment]
@franckc 
it's done now

**Comment by franckc** [comment]
Merging this in master now ! :)

**Pull request merged**